Frontend/src/pages/confirmation/index.tsx

```diff
@@ -25,6 +25,7 @@ export default function Confirmation () {
   
   const submit = async (e: SyntheticEvent) => {
     e.preventDefault();
+    console.log(token)
```

Comment:
can you please add for example "token = {Token}" not just logging it (because we would get later logging too many things and we don't know which is for what

Reply:
This will be removed

Reply:
Okay just put todo about it to not forget